docs/detections/add-exceptions.asciidoc

```diff
@@ -53,7 +53,12 @@ image::images/rule-exception-tab.png[Detail of rule exceptions tab]
 .. Click *Create shared exception list* -> *Create exception item*. 
 --
 
-. In the *Add rule exception* flyout, name the exception and add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
+. In the *Add rule exception* flyout, do the following: 
+
+.. Name the exception. 
+.. Add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
++
+NOTE: Exception conditions are auto-populated with relevant alert data when you create a new exception from the alert's details flyout or from the Alert table.
```

Comment:
Great description Nastasha!.
It is indeed accurate that there is no need to mention the "Highlighted fields" since we only utilize the relevant data from them only.

Reply:
Note to self that I might need to expand this note to include the comment that's also auto-generated when users create a new exception from the Alerts table or flyout.

Reply:
```suggestion
NOTE: When you create a new exception from the alert's details flyout or from the Alert table, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.
```